### PR TITLE
contracts_configuration: Add documentation on deployment tracking.

### PR DESCRIPTION
**Pull request by cryptomental**
Refs: https://github.com/embark-framework/embark/issues/938

### CODE REVIEW

File: source/docs/contracts_configuration.md [hunk 1]
```diff
@@ -248,6 +248,25 @@ module.exports = {
 }
 </code></pre>
 
+### Deployment tracking
+
+Contract deployment tracking mechanism prevents deployment if contract code was already deployed.
+It is possible to disable this feature for a given contract by setting `track` configuration field to false. This way the contract will be always deployed.
```
[the remark this answers is not in the document]
Comment (cryptomental) [reply source/docs/contracts_configuration.md]: @PascalPrecht yessir! Commit amended, false is put into backticks now. :)

[the remark this answers is not in the document]
Comment (PascalPrecht) [reply source/docs/contracts_configuration.md]: You rock, thanks!

**Comment by PascalPrecht** [comment]
Okay, so this is in a LGTM state. We could merge this but can only release when Embark also releases this feature. 

Another way to go about this is to merge those things in a separate branch so we can keep releasing docs updates unrelated to releases in the meantime.

@iurimatias wdyt?

**Comment by jrainville** [comment]
@PascalPrecht , don't worry, develop of embark-site is "synced" with embark's develop. So we only release (publish) the develop website when we publish embark develop.

File: source/docs/contracts_configuration.md [hunk 2]
```diff
@@ -248,6 +248,25 @@ module.exports = {
 }
 </code></pre>
 
+### Deployment tracking
+
+Contract deployment tracking mechanism prevents deployment if contract code was already deployed.
```
[the remark this answers is not in the document]
Comment (cryptomental) [reply source/docs/contracts_configuration.md]: Hi @michaelsbradleyjr , 'me broken English' ;) Could you please refresh the page and have a look again? I updated the commit.

**Pull request merged**